src/Test/IntegrationCase.php

```diff
@@ -95,6 +103,10 @@ public function hasInputCode()
         return null !== $this->inputCode;
     }
 
+    public function getConfig()
+    {
+        return $this->config;
+    }
```

Comment:
actually not. see #2287 
with this bug there is priority issue between method_separation and no_extra_blank_lines fixers.